Document/0x04f-Testing-Network-Communication.md

```diff
@@ -70,75 +73,34 @@ Wireshark offers a GUI and is more straightforward if you are not used to the co
 
 **Network Setup**
 
-To be able to get a man-in-the-middle position your machine should be in the same wireless network as the mobile phone and the gateway it communicates to. Once this is done you need the following information:
-
-- IP address of mobile phone
-- IP address of gateway
+To be able to get a man-in-the-middle position your machine should be in the same wireless network as the mobile phone and the gateway it communicates to. Once this is done you need the IP address of mobile phone.
 
-#### ARP Poisoning with Ettercap
-
-Start ettercap with the following command and replace the first IP addresses with the network gateway in the wireless network and the second one with the one of your mobile device.
-
-```shell
-$ sudo ettercap -T -i en0 -M arp:remote /192.168.0.1// /192.168.0.105//
-```
+#### ARP Poisoning with bettercap
 
-On the mobile phone start the browser and navigate to example.com, you should see output like the following:
+Start your preferred network analyzer tool first, then start bettercap with the following command and replace the IP address 192.168.0.103 with the target you want to execute the MITM attack against.
```

Comment:
I would use template IPs instead of actual IP addresses. Like xx.xx.xx.xx

Reply:
makes sense. thanks